src/Dockerfile

```diff
@@ -55,6 +55,9 @@ COPY root/usr/libexec/fix-permissions /usr/libexec/fix-permissions
 {% if spec.repo_enable_reason %}
 {{ spec.repo_enable_reason }}
 {% endif %}
+{% if spec.prod == "c10s" %}
+RUN INSTALL_PKGS="rsync tar gettext bind-utils nss_wrapper {{ spec.pkgs }}" && \
```

Comment:
We can use `nss_wrapper-libs` (in all containers, all distros as it's included since rhel-8.8.0). It's a separate sub-package without unnecessary footprint and we're ok with the `*.so` file only.

But we can make this task an all-containers effort, and not block this PR on that -- I created https://issues.redhat.com/browse/RHEL-57583 to track it.

Reply:
Fixed by https://github.com/sclorg/postgresql-container/pull/587/commits/44b75b9b0653b0a7537caade4d5b64cf05ac1e01